components/input/style/token.ts

```diff
@@ -72,6 +72,36 @@ export interface SharedComponentToken {
    * @descEN Background color when the input box is activated
    */
   activeBg: string;
+  /**
+   * @desc 内容字体大小
+   * @descEN Font size of content
+   */
+  contentFontSize: number;
```

Comment:
```suggestion
  inputFontSize: number;
```
content 在 Input 里没啥意义，用 input 指代输入框好点

Reply:
我看有好多 diff，是不是不需要 lineHeight？